Artur_Aghajanyan/Python/25_05_2021/pascal.py

```diff
@@ -0,0 +1,13 @@
+#from __future__ import print_function
+
+n = input("Row: ")
+
+def printPascal(n):
+    for line in range(1, n + 1):
+        el = 1
+        for i in range(1, line + 1):
+            print(el, end = " ");
+            el = int(el * (line - i) / i);
+        print("")
+
+printPascal(int(n))
```

Comment:
1. coding style -> function name
2. validation for big number
![image](https://user-images.githubusercontent.com/80816144/119569458-d0896300-bdbf-11eb-89df-b63cfd9f44c4.png)

3.